components/trezor_bridge/resources/BUILD.gn

```diff
@@ -0,0 +1,39 @@
+import("//brave/components/common/typescript.gni")
+
+transpile_web_ui("trezor_bridge_ui") {
+  entry_points = [
+    [
+      "trezor_bridge",
+      rebase_path("trezor_bridge.ts"),
+    ],
+    [
+      "trezor/trezor",
+      rebase_path("trezor/iframe.ts"),
+    ],
+  ]
+
+  resource_name = "trezor_bridge"
+}
+
+pack_web_resources("generated_resources") {
+  resource_name = "trezor_bridge"
+  output_dir = "$root_gen_dir/brave/components/trezor_bridge/resources"
+  deps = [
+    ":data",
```

Comment:
I don't think this is right - this target doesn't depend on `:data` because it's not the one to include the data files. The grit target which uses the copied files is the one which should depend on it.

Reply:
done